docs/src/index.md

```diff
@@ -1,4 +1,31 @@
-# How to guide
+# Thermodynamics
+
+## Principal design
+
+This package uses an abstraction that leverages the idea of a thermodynamic state:
+
+ - given two (or more) independent intrinsic thermodynamic properties, we can establish a thermodynamic state and
+ - given a thermodynamic state, we can compute any thermodynamic property
+
+ 
+## Simple example
+
+For our example, we first load packages, and create a set of thermodynamic parameters, using a convenience constructor, offered through [ClimaParams.jl](https://github.com/CliMA/ClimaParams.jl). Then we create a thermodynamic state using density, liquid-ice potential temperature, and total specific humidity. Finally, we compute air temperature from the thermodynamic state.
+
+```@example
+using ClimaParams # needed in environment to load convenience parameter struct wrappers
+import Thermodynamics as TD
+params = TD.Parameters.ThermodynamicsParameters(Float64);
```

Comment:
might be nice to show the parameters too here, for me knowing what exactly the parameters are is very clarifying to understand how the physical calculations are made subsequently

Reply:
Yeah, I'm running out of time right now. Let's merge and we can expand on this.